setup.cfg

```diff
@@ -1,10 +1,9 @@
-[nosetests]
-nocapture=1
-cover-package=zope.annotation
-cover-erase=1
-with-doctest=0
-where=src
-
 [aliases]
 dev = develop easy_install zope.annotation[testing]
 docs = easy_install zope.annotation[docs]
+
+[bdist_wheel]
+universal = 1
+
+[zest.releaser]
+with create-wheel = yes
```

Comment:
In none of the recent packages I've worked on (e.g., zope.app.apidoc) have I needed to add a `[zest.releaser]` section in `setup.cfg` in order to have wheels built and uploaded. I thought at first that maybe that happened because of the `[bdist_wheel]` section, but I accidentally forgot that once on [zope.app.zcmlfiles](https://pypi.python.org/pypi/zope.app.zcmlfiles/4.0.0) and `fullrelease` happily built and uploaded a `py2` wheel.

Investigating, I now realize that my own user `~/.pypirc` is where the `create-wheel` setting is. Apparently I added that a long time ago and forgot about it and in doing so neglected to add it explicitly to all those packages 😢 

(Maybe having zest.releaser check for a `[bdist_wheel] universal = 1` setting as a fallback is a good feature request?)